src/tranquilo/wrap_criterion.py

```diff
@@ -66,3 +64,24 @@ def wrapper_criterion(eval_info):
         )
 
     return wrapper_criterion
+
+
+def process_batch_evaluator(batch_evaluator="joblib"):
+    batch_evaluator = "joblib" if batch_evaluator is None else batch_evaluator
+
+    if callable(batch_evaluator):
+        out = batch_evaluator
+    elif isinstance(batch_evaluator, str):
+        if batch_evaluator == "joblib":
+            from estimagic.batch_evaluators import joblib_batch_evaluator as out
+        elif batch_evaluator == "pathos":
+            from estimagic.batch_evaluators import pathos_mp_batch_evaluator as out
```

Comment:
Do we need to remove this estimagic import as well?

Reply:
Imports inside functions are not a problem for circular imports. That's why I wrote this function instead of importing and calling estimagic's `process_batch_evaluator` function.